server/utils/middleware.js

```diff
@@ -29,6 +29,18 @@ export function ifNoUser401(req, res, next) {
   return res.status(401).end();
 }
 
+export function isNamed(req, res, next) {
+  if (req.user.name === '') {
+    return res.status(200).send(`We need your name so we can put it on your
+      certificate.<a href="https://github.com/settings/profile"> Add your name
```

Comment:
You have a space before the word `Add`. Move it before the `a` tag:
`...certificate. <a href="https://github.com/settings/profile">Add your nam...`